src/app/src/test/kotlin/org/imperial/mrc/hint/database/VersionRepositoryTests.kt

```diff
@@ -278,7 +278,6 @@ class VersionRepositoryTests
         assertThat(newVersion.note).isEqualTo(null)
     }
 
-
```

Comment:
Which change ?

Reply:
A blank line was seemingly removed from this file? Hence it has been pulled into this PR. I know it is a really small issue but it's something that can usually be spotted by having a final look through the PR on GitHub itself before you assign it to reviewers. This is a good habit in case there are other similar but more significant issues (i.e. files committed that shouldn't be).